src/examples/colors/generate_wcs_languages.py

```diff
@@ -222,69 +239,95 @@ def meaning_distance(center, point):
 
         color_names = temp_color_terms
 
-        #Create a zeroed np array of the number of colors x the number of major color terms within the language
-        expression_meanings = np.array([[0.0 for _ in range(len(color_universe.referents))] for _ in range(len(major_color_terms))])
+        # Create a zeroed np array of the number of colors x the number of major color terms within the language
+        expression_meanings = np.array(
+            [
+                [0.0 for _ in range(len(color_universe.referents))]
+                for _ in range(len(major_color_terms))
+            ]
+        )
 
-        #Print the expression array dimensions
-        #print(f"Expression array dimensions: {expression_meanings.shape}")
+        # Print the expression array dimensions
+        # print(f"Expression array dimensions: {expression_meanings.shape}")
 
-        #For each major color chip in the language, add the Gaussian meaning row corresponding to that chip to the row for that color term.
+        # For each major color chip in the language, add the Gaussian meaning row corresponding to that chip to the row for that color term.
         for color in language_colors:
-            color_meaning_dist = meaning_dists[list(color_universe.referents).index(referents_by_color_code[color])]
-            for major_term, major_term_count in language_colors_to_expressions[language_code][color].items():
+            color_meaning_dist = meaning_dists[
+                list(color_universe.referents).index(referents_by_color_code[color])
+            ]
+            for major_term, major_term_count in language_colors_to_expressions[
+                language_code
+            ][color].items():
                 if major_term in major_color_terms:
-                    #Option: Take the maximum
-                    expression_meanings[major_color_terms.index(major_term)] = np.maximum(expression_meanings[major_color_terms.index(major_term)], color_meaning_dist)
-                    #Option: Add just one value, for the color represented
+                    # Option: Take the maximum
+                    expression_meanings[major_color_terms.index(major_term)] = (
```

Comment:
similarly, here and above, `.index()` is very expensive; there should be a way to use `enumerate` or something similar with `major_color_terms` in conjunction with the counter I think

Reply:
I also don't think we should need the `color_meaning_dist` at all at this stage: that one is capturing the perceptual uncertainty, not the empirical frequency, which is what we want for the nat langs

Reply:
Can set up some sort of enumerated, indexed list so that we can avoid calling `index()` too many times.

Reply:
The `color_meaning_dist` was intended to allow closer colors to have a higher probability based off the Gaussian distribution, instead of being classified as a hard binary, allowing for a softer partition of the color space. Can remove this if necessary.